### DISCUSSION TITLE
PaddleX/latest/index

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#2446] [General]
# PaddleX/latest/index

https://paddlepaddle.github.io/PaddleX/latest/index.html

**Comment by G-lin** [comment]
OCR推理慢，怎么解决，11s识别一次

**Reply by cuicheng01** [reply]
请问你是如何识别的呢？方便的话，请给出详细的测试demo～

**Comment by cuicheng01** [comment]
请问你是如何识别的呢？方便的话，请给出详细的测试demo～

**Comment by MingJ-Zhang** [comment]
如何设置只使用第一个GPU？

**Reply by cuicheng01** [reply]
设置`device`为`gpu:0`

**Comment by dimy1986** [comment]
是不是只能支持gpu？我安装的cpu版本 PaddlePaddle，但是运行报错，ValueError: invalid literal for int() with base 10: 'Failed to initialize NVML: Unknown Error'
from paddlex import create_pipeline

pipeline = create_pipeline(pipeline="OCR")
output = pipeline.predict(
    input=r"D:\***\法人身份证正面.jpg",
    device="cpu",
    use_doc_orientation_classify=False,
    use_doc_unwarping=False,
    use_textline_orientation=False,

)
for res in output:
    res.print()

**Comment by likou123** [comment]
使用pip install paddlex==3.0.0rc1安装paddleX会在wheel安装panda的时候报错，具体为卡在Collecting pandas<=1.5.3 (from paddlex==3.0.0rc1)这部分，报错显示  
ERROR: Failed building wheel for pandas
Failed to build pandas
ERROR: Failed to build installable wheels for some pyproject.toml based projects (pandas) 
之前还可以用https://paddle-model-ecology.bj.bcebos.com/paddlex/whl/paddlex-3.0.0b2-py3-none-any.whl这种改内置配置文件中的包版本来解决
现在应该怎么办啊
我的设备为：python12.0+cu126

**Comment by wangtianxiong1995** [comment]
pandas<=1.5.3依赖版本太低了，和现有的包冲突该怎么办

**Reply by wangtianxiong1995** [reply]
提到>=2.0.0好么

**Comment by cody111115** [comment]
请问一下如何使用paddlex使用自己训练的模型，以及部署到rk3588/rk3576

**Reply by Bobholamovic** [reply]
paddlex可以通过配置文件更换模型，详情请参考文档。部署到rk3588我们暂时没有尝试过，如果你在尝试过程中遇到问题，欢迎提问。

**Comment by ganzhiming** [comment]
请教一个问题，我用PaddleX3.2源码本地训练自定义数据集，用的模型是:Cascade-FasterRCNN-ResNet50-vd-SSLDv2-FPN，为什么训练输出的的inference模型的后缀是inference.json格式的，不是更新的.pdmodel格式的？

**Comment by shuaigege-coder** [comment]
你好，我用通过paddlex进行了服务化部署，PPOCRV5，我想请问一下在公司内网的情况下，如何配置代理，要不然传递过来的URL链接都没办法下载下来
